src/metabase/api/search.clj

```diff
@@ -97,25 +96,7 @@
                                 :search-native-query                 search_native_query
                                 :search-string                       q
                                 :table-db-id                         table_db_id
-                                :verified                            verified}))
-        duration             (- (System/currentTimeMillis) start-time)
-        has-advanced-filters (some some?
-                                   [models created_by created_at last_edited_by
-                                    last_edited_at search_native_query verified])]
-    (when (contains? #{"search-app" "search-bar" "command-palette"} context)
-      (snowplow/track-event! ::snowplow/new-search-query api/*current-user-id*
-                             {:runtime-milliseconds duration
-                              :context              context})
-      (when has-advanced-filters
-        (snowplow/track-event! ::snowplow/search-results-filtered api/*current-user-id*
-                               {:runtime-milliseconds  duration
-                                :content-type          models
-                                :creator               (some? created_by)
-                                :creation-date         (some? created_at)
-                                :last-editor           (some? last_edited_by)
-                                :last-edit-date        (some? last_edited_at)
-                                :verified-items        (some? verified)
-                                :search-native-queries (some? search_native_query)})))
+                                :verified                            verified}))]
     results))
```

Comment:
nit - no need for the `results` binding anymore, since there's no extra snowplow logic before we return the results. Can you just make the `search` call the last form in this function (i.e. return the results directly)?